### ISSUE TITLE
v0.6.2 release

### ISSUE THREAD

**Issue by bpatel2107** [bpatel2107/pyrokinetics#316]
I think we are well overdue for a new release! Wait for the following to be merged and then we can crack on with a release

- [ ] #301
- [ ] #311 
- [x] #315

**Comment by LiamPattinson** [comment]
I agree, now looks like a good time! It would definitely be good to have a release with the `pygacode` alternative you wrote, as currently it can lead to `pip install pyrokinetics` failing on some systems.

**Comment by LiamPattinson** [comment]
I just raised PR https://github.com/pyro-kinetics/pyrokinetics/pull/340 which updates the PyPI publishing workflow to the new standards. All three of the bullet points in the top post have been achieved, so is there anything else that we need before the next release? We're adding quite a lot and there are some changes to the user interface, so I'd argue this should be v0.7.0 instead of v0.6.2.

Should we also organise a development meeting sometime soon? It'd be good to catch up and discuss what our priorities should be beyond the next release.

**Comment by LiamPattinson** [comment]
Cheers for merging PR https://github.com/pyro-kinetics/pyrokinetics/pull/345! Are there any other tasks that need doing before releasing v0.7.0?

**Comment by bpatel2107** [comment]
Technically no but I am very close to making a big change on `GKOutput`. But that can probably be for v0.7.1 but when that is merged I'll want to do another release basically immediately

**Comment by LiamPattinson** [comment]
No issue with doing another release after new features have been added!

I'm personally working on a fix to Issues https://github.com/pyro-kinetics/pyrokinetics/issues/326 and https://github.com/pyro-kinetics/pyrokinetics/issues/337, and I'm hoping to have a demonstration ready tomorrow or early next week. It includes a rewrite and merging of `SimulationNormalisation` and `ConventionNormalisation` into a single simplified class, and rethinks how units are assigned and updated. However, the full implementation would affect a large proportion of the project, and will need to be split over multiple issues and pull requests (it would probably be best to manage using GitHub Projects). The sum total of those updates would probably require a v0.8.0 release, but if I can compartmentalise each of the prerequisite steps well enough, it may also result in versions 0.7.2, 0.7.3, etc.
